src/Avalonia.Base/Animation/Spring.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.ComponentModel;
+using System.Globalization;
+using Avalonia;
+using Avalonia.Utilities;
+
+namespace Avalonia.Animation;
+
+/// <summary>
+/// Determines how an animation is used based on spring formula.
+/// </summary>
+[TypeConverter(typeof(SpringTypeConverter))]
```

Comment:
Note that XAML compiler doesn't need a converter if type has a static Parse method